var/spack/repos/builtin/packages/gromacs/package.py

```diff
@@ -718,30 +718,30 @@ def cmake_args(self):
         else:
             # we rely on the fftw-api@3
             options.append("-DGMX_FFT_LIBRARY=fftw3")
-            if "^amdfftw" in self.spec:
+            if self.spec.satisfies("^amdfftw"):
```

Comment:
```suggestion
            if self.spec.satisfies("^[virtuals=fftw-api] amdfftw"):
```
@mabraham @eirrgang @junghans Please verify this addition of explicit virtuals for `gromacs ^amdfftw`, per https://spack.readthedocs.io/en/latest/basic_usage.html#explicit-binding-of-virtual-dependencies.

Reply:
That looks good.

Reply:
I don't have a button to accept the suggested change, however